README.md

```diff
@@ -36,6 +36,22 @@ Create a `NotificationRequest` providing two documents: the act to notify and th
       - `payment` referencing one file uploaded previously
    - the `documents` field providing one file uploaded previously
 
+### TC-SEND-01bis
+Create a `NotificationRequest` providing only the act to notify.
+
+1. Request an upload slot
+2. Consume the upload slot uploading the document to notify (use the information provided in the previous request).
+3. Create a notification request providing the following information:
+    - `physicalCommunicationType` filled with `REGISTERED_LETTER_890` value
+    - the `recipients` field providing:
+        - `taxId`
+        - `digitalDomicile`
+        - `physicalAddress`
+        - `payment` referencing one file uploaded previously
```

Comment:
The payment field shouldn't be filled here; the use case is about sending a document without any payment, right?